src/bindings.js

```diff
@@ -278,18 +278,23 @@ export class ComponentBinding extends Binding {
     let bindingRegExp = view.bindingRegExp()
 
     if (this.el.attributes) {
-      this.el.attributes.forEach(attribute => {
+      let attribute = null
```

Comment:
why do you create it here? I believe you can remove this line and move `let` on 283